.github/workflows/docs.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and deploy Jekyll documentation site
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  RUBY_VERSION: 2.7
+
+jobs:
+  deploy_docs:
+    if: "!contains(github.event.commits[0].message, '[ci skip]')"
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+      - name: Setup cache for Bundler
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-bundler-${{ env.RUBY_VERSION }}-${{ hashFiles('Gemfile') }}-${{ hashFiles('jekyll.gemspec') }}
```

Comment:
I think the `Gemfile.lock` file would be the better thing to hash here. Dependencies can be updated without updating the Gemfile, and I'd feel like we'd want to invalidate the cache in those cases. Maybe this doesn't matter for building the site though. I'm fine if we want to leave it as is.